lib/applications/src/launcher.cpp

```diff
@@ -19,20 +19,25 @@
  */
 std::string getFormatedDate()
 {
-    uint16_t day_ = GSM::days;
-    uint16_t day = GSM::days;
-    uint16_t month = GSM::months;
-    uint16_t year = GSM::years;
+    int16_t day_ = Gsm::Time::getDay();
+    int16_t day = Gsm::Time::getDay();
+    int16_t month = Gsm::Time::getMonth();
+    int16_t year = Gsm::Time::getYear();
+
+    if(day_ == -1) day_ = 1;
```

Comment:
day_ pas utilisé ici
```suggestion
    int16_t day = Gsm::Time::getDay();
    int16_t month = Gsm::Time::getMonth();
    int16_t year = Gsm::Time::getYear();
```